tensorflow/compiler/jit/xla_device.h

```diff
@@ -126,8 +126,8 @@ class XlaDevice : public LocalDevice {
     // the logical on-device shape without padding is used.
     PaddedShapeFn padded_shape_fn;
 
-    // Set of devices to use. This controls which of the devices given type in
-    // the system will have resources allocated for. For GPUs this will be
+    // Set of devices to use. This controls which of the devices on the given
+    // platform resources will have resources allocated. For GPUs this will be
```

Comment:
> This controls which of the devices on the given
> platform resources will have resources allocated.

Should the first "resources" be removed?

Reply:
Done.